### PR TITLE
Find characters from english alphabet

### PR DESCRIPTION
**Pull request by ann-kolosova**
Regex solution that finds characters from english alphabet in input.txt file and write result in output.txt file

### CODE REVIEW

File: Lesson09_Task01.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+"""
+
+import re
+
+try:
+    with open('input.txt', 'r') as input_txt:
+        data = input_txt.read()
+
+    with open('output.txt', 'w') as output_txt:
+        for let in re.finditer('[a-zA-Z]', data):
```
Comment (antohaUa) [inline Lesson09_Task01.py:24]: better name as -> `for curr_match in re.finditer('[a-zA-Z]', data)`  cause technically this is match object

File: Lesson09_Task01.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            output_txt.write(f'<{let.group(0)}> -> pos {let.start()}\n')
+
+except Exception as e:
+    print(e)
```
Comment (antohaUa) [inline Lesson09_Task01.py:28]:
```suggestion
except Exception as g_exc:
    print(g_exc)
```

File: Lesson09_Task01_Alternative.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# #######################
+"""
+
+import re
+
+try:
+    with open('input.txt', 'r') as input_txt:
+        data = input_txt.read()
+        eng_letters = re.findall('[a-zA-Z]', data)
+        letter_pos = [(letter, data.index(letter)) for letter in eng_letters]
```
Comment (antohaUa) [inline Lesson09_Task01_Alternative.py:23]: This won't work for duplicate characters. But this is alternative solution not primary ))

**Review by antohaUa — APPROVED** [review]

File: Lesson09_Task01.py (hunk 2 above)
Comment (ann-kolosova) [reply Lesson09_Task01.py]: fixed

File: Lesson09_Task01_Alternative.py (hunk 3 above)
Comment (ann-kolosova) [reply Lesson09_Task01_Alternative.py:23]: file with the incorrect solution was removed

**Pull request merged**